test/api-digester/Outputs/stability-stdlib-abi.swift.expected

```diff
@@ -6,6 +6,7 @@ Func MutableCollection._introSort(within:by:) has been removed
 Func MutableCollection._introSortImpl(within:by:depthLimit:) has been removed
 Func MutableCollection._siftDown(_:within:by:) has been removed
 Func MutableCollection._sort3(_:_:_:by:) has been removed
+Func _createStringTableCache(_:) has been removed
```

Comment:
The ABI list is unsorted now, can you group your entries together down the bottom?

Reply:
ok, done